infrastructure/src/main/java/org/corfudb/infrastructure/DataStore.java

```diff
@@ -1,9 +1,6 @@
 package org.corfudb.infrastructure;
 
-import com.github.benmanes.caffeine.cache.CacheWriter;
-import com.github.benmanes.caffeine.cache.Caffeine;
-import com.github.benmanes.caffeine.cache.LoadingCache;
-import com.github.benmanes.caffeine.cache.RemovalCause;
+import com.github.benmanes.caffeine.cache.*;
```

Comment:
I think the consensus was that we will have expanded imports instead of .*. Not sure if I remember this correctly